Automattic/pocket-casts-android
# Prepare database for syncing additional podcast settings

## Description

Our backend is now ready to sync all podcast settings. This PR adds all missing columns to our database that will allow syncing them.

## Testing Instructions

1. Install the app from an older commit. For example daa1e07327b0199f0b068d732470fa6f54445cac.
2. Have some podcasts that you're subscribed to.
3. Checkout `update/podcast-sync-db-migration` branch. Make sure that it is up-to-date.
4. Install the app.
5. Open the app and make sure that app didn't crash and you can still see all of your podcasts without a need to sync the account.

## Checklist
 
#### I have tested any UI changes...
